addons/sale_product_configurator/static/src/js/product_configurator_widget.js

```diff
@@ -322,28 +316,28 @@ ProductConfiguratorWidget.include({
             };
 
             if (product.no_variant_attribute_values) {
-                var default_product_no_variant_attribute_values = [];
-                _.each(product.no_variant_attribute_values, function (attribute_value) {
-                        default_product_no_variant_attribute_values.push(
-                            [4, parseInt(attribute_value.value)]
+                var defaultProductNoVariantAttributeValues = [];
```

Comment:
`VeryInterestingDiff`!
`or_is_it`?

Reply:
I took the opportunity to pass the js through a more thorough linter check, because in any case this code is mine, but i can drop it if you really don't want it :D.